recipes/deeptools/meta.yaml

```diff
@@ -0,0 +1,45 @@
+package:
+  name: deeptools
+  version: '2.0.0'
+
+source:
+  fn: deepTools-2.0.0.tar.gz
+  url: https://pypi.python.org/packages/source/d/deepTools/deepTools-2.0.0.tar.gz
+  md5: 09821e73f56ddb744a35d01173c6eb29
+
+build:
+  number: 0
+  skip: True # [not py27]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.8.0
```

Comment:
Ah, it includes the numpy version in the tarball name. I guess since it uses the most recent version available then the >=1.8.0 would end up being true anyway.